src/VisualStudio.Roslyn.SDK/Roslyn.SDK/ProjectTemplates/CSharp/CodeRefactoring/Vsix/Deployment.csproj

```diff
@@ -1,23 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project>
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
```

Comment:
Generally a fan of this. Are there any known issues with VSIX projects in sdk-style projects? in the past there were bugs with the VSSDK around doing this.

Reply:
There are known issues, but they are addressed by the form of this project.

1. There are two explicit (linked) workarounds in the project file
2. The explicit `<Import>` elements are required to have everything defined in the correct order (i.e. the simplified `<Project Sdk="...">` form will not work for VSIX projects)